### PR TITLE
added aria hidden tag to div surrounding svg

### PR DESCRIPTION
**Pull request by MassivaM**
## Changes in this pull request
Added an aria-hidden=true tag surrounding the svg element so keyboard readers skip decorative icons 

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] All applicable changes have been documented
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment

### CODE REVIEW

**Comment by MassivaM** [comment]
@emensch You mentioned wanting to think about this more so I'm making this PR as a starting point , I'm not sure how to add the tag directly in the svg tag if possible

File: packages/c2pa-wc/src/components/Icon/Icon.ts [hunk 1]
```diff
@@ -40,28 +40,42 @@ export class Icon extends LitElement {
...
       icon: html`<cai-icon-adobe-stock></cai-icon-adobe-stock>`,
     },
-    { pattern: /adobe/i, icon: html`<cai-icon-adobe></cai-icon-adobe>` },
     {
+      name: 'adobe',
+      pattern: /adobe/i,
+      icon: html`<cai-icon-adobe></cai-icon-adobe>`,
+    },
+    {
+      name: ' behance',
```
Comment (dkozma) [inline packages/c2pa-wc/src/components/Icon/Icon.ts]:
```suggestion
      name: 'behance',
```

File: packages/c2pa-wc/src/components/Icon/Icon.ts [hunk 2]
```diff
@@ -40,28 +40,42 @@ export class Icon extends LitElement {
 
   static readonly matchers = [
     {
+      name: 'photoshop',
```
Comment (dkozma) [inline packages/c2pa-wc/src/components/Icon/Icon.ts]: What is the addition of `name` here trying to support?

**Review by dkozma — APPROVED** [review]
LGTM

**Pull request merged**